MonoGame.Framework/Utilities/OS.cs

```diff
@@ -0,0 +1,46 @@
+﻿// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+namespace MonoGame.Utilities
+{
+    /// <summary>
+    /// Defines the operating system.
+    /// </summary>
+    public enum OS
```

Comment:
Well it has to be maintained.

If you want to check at runtime for Windows/Mac/Linux, it's as simple as:

```
            if (Path.DirectorySeparatorChar == '/')
            {
                if (Directory.Exists("/Library"))
                {
                    return "MacOS";
                }

                return "Linux";
            }

            return "Windows";
```

anyone who needs to detect it at runtime can do it.

It just seems really arbitrary to be adding another platform detection class to MonoGame.